### DISCUSSION TITLE
Is there a dashboard for Conductor metrics?

### DISCUSSION THREAD

**Discussion by emr17** [Netflix/conductor-community#187] [General]
I have activated the metrics for the conductor through prometheus, but I want to visualize these metrics. I saw that grafana was used for this, but I could not find a dashboard prepared according to metrics. There is a dashboard that was prepared 3 years ago, but it does not work, I guess because it is not up to date. Can you help if there is a currently working dashboard?

**Comment by DukeDai** [comment]
> There is a dashboard that was prepared 3 years ago,
where is it? if there is up-to-date template, we could make one based on the one which does not work.

**Reply by emr17** [reply]
https://gist.github.com/marosmars/40eeea1645b4c281d6aeb645b02608c4    
it doesnt work now
